schema/me/conversations.js

```diff
@@ -0,0 +1,68 @@
+import date from '../fields/date';
+import impulse from '../../lib/loaders/impulse';
+import gravity from '../../lib/loaders/gravity';
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLList,
+  GraphQLInt,
+} from 'graphql';
+const { IMPULSE_APPLICATION_ID } = process.env;
+
+const ConversationType = new GraphQLObjectType({
+  name: 'ConversationType',
+  description: 'A conversation.',
+  fields: {
+    id: {
+      description: 'Impulse id.',
+      type: GraphQLString,
+    },
+    inquiry_id: {
+      description: 'Gravity inquiry id.',
+      type: GraphQLString,
+    },
+    from_id: {
+      type: GraphQLString,
+    },
+    from_type: {
+      type: GraphQLString,
+    },
+    from_email: {
+      type: GraphQLString,
+    },
+    to_id: {
+      type: GraphQLString,
+    },
+    to_type: {
+      type: GraphQLString,
+    },
+    buyer_outcome: {
+      type: GraphQLString,
+    },
+    buyer_outcome_at: date,
+    initial_message: {
+      type: GraphQLString,
+    },
+  },
+});
+
+export default {
+  type: new GraphQLList(ConversationType),
+  decription: 'Converations for the user. First a token from gravity is requested.',
```

Comment:
Rather than state what the internals do, this could maybe delve a bit more into what conversations are so that a reader of the docs knows what they need it for.